wercker.yml

```diff
@@ -54,6 +54,23 @@ build:
           echo "build is skipped ..."
         fi
 
+  - script:
+      name: mvn clean verify
+      code: |
+        if [[ $SKIP_CI == 'false' ]]; then
+          cd sevntu-checks
+          mkdir -p .ci-temp
+          cd .ci-temp
+          git clone -b issue-10100 https://github.com/nmancus1/checkstyle.git
+          cd checkstyle
+          mvn -e clean install -Pno-validations
+          cd ../..
+          mvn clean verify -Dcheckstyle.eclipse-cs.version=8.44-SNAPSHOT
```

Comment:
Temporary hack to show that we can pass `mvn clean verify` with checkstyle version from base repo PR.